packages/agoric-cli/src/commands/ec.js

```diff
@@ -239,6 +265,11 @@ export const makeEconomicCommiteeCommand = (_logger, io = {}) => {
       const utils = await makeRpcUtils({ fetch });
       const { readLatestHead } = utils;
 
+      if (!(opts.sendFrom || opts.offerId))
```

Comment:
`opts.offerId` should be something like `previousOffer`, yes? Is there a desire to bring back the `--previousOffer` option rather than automatically looking it up?

`opts.offerId` is the id of the new place-my-vote offer

Reply:
Typo thanks

Reply:
I read this on my phone and misremembered the context. It's more than a typo. Innocuous since `sendFrom` is required, but validating that per se is the real fix: https://github.com/Agoric/agoric-sdk/pull/7721/commits/4f49cf448bf64592b692a5e6e087e9affe8a61eb